jenkins-scripts/lib/project-default-devel-homebrew-amd64.bash

```diff
@@ -61,9 +61,14 @@ echo '# BEGIN SECTION: setup the osrf/simulation tap'
 brew tap osrf/simulation
 echo '# END SECTION'
 
-if [[ -n "${PULL_REQUEST_URL}" ]]; then
-  echo "# BEGIN SECTION: pulling ${PULL_REQUEST_URL}"
-  brew pull ${PULL_REQUEST_URL}
```

Comment:
> The brew pull command has been deprecated

So there's no use for `PULL_REQUEST_URL` anymore? Should it be removed?

Reply:
it's not used in this job, but I think it is used in a downstream job that updates the bottle hash `sha256` values

Reply:
confirmed that it is still in use in the downstream job

* https://github.com/ignition-tooling/release-tools/blob/9bfe8c23e9801a60da815132395626a508cb4860/jenkins-scripts/dsl/brew_release.dsl#L224-L235
* https://github.com/ignition-tooling/release-tools/blob/9bfe8c23e9801a60da815132395626a508cb4860/jenkins-scripts/dsl/brew_release.dsl#L257-L259

Reply:
> confirmed that it is still in use in the downstream job

If I read correctly the DSL code:
```dsl
  parameters { 
        currentBuild() 
          predefinedProp("PULL_REQUEST_URL", "https://github.com/osrf/homebrew-simulation/pull/\${ghprbPullId}") 
}
```

Looks to me like we are overriding `PULL_REQUEST_URL` value to the base URL + `ghprbPullId` instead of using the current one in this script (injected by `currentBuild()`.

If we remove `PULL_REQUEST_URL` from here, is there anyway of launching the job manually? Can that be done without these changes? Maybe is not a supported use case.

Reply:
ok, maybe some of this logic is left over from when `release.py` could trigger a bottle build and would pass `PULL_REQUEST_URL`. Now we only start bottle builds from GitHub comments. I'll see if there's something that should be removed

Reply:
> If we remove PULL_REQUEST_URL from here, is there anyway of launching the job manually? Can that be done without these changes? Maybe is not a supported use case.

it's not easy to start the "bottle hash updater job" manually since it needs artifacts from a bottle build

I looked and I don't think there's anything more to be changed